### PR TITLE
fix(ui5-bar): add rtl mode supported

### PR DESCRIPTION
**Pull request by Todor-ads**
fix(ui5-bar): add rtl mode supported

### CODE REVIEW

**Review by NHristov-sap — APPROVED** [review]

File: packages/fiori/src/Bar.js [hunk 1]
```diff
@@ -164,10 +166,21 @@ class Bar extends UI5Element {
 			root: {
 				"ui5-bar-root": true,
 				"ui5-bar-root-shrinked": this._shrinked,
+				"ui5-bar-root-rtl": this._isRTL,
 			},
 		};
 	}
 
+	get _isRTL() {
+		return this.effectiveDir === "rtl";
```
Comment (ilhan007) [inline packages/fiori/src/Bar.js:175]: you can just use effectiveDir in the template as value of dir attribute and then use the dir attribute in the CSS

If you do that, you can remove most of the code:
- "ui5-bar-root-rtl": this._isRTL,

- get _isRTL() {
		return this.effectiveDir === "rtl";
}

**Review by ilhan007 — CHANGES_REQUESTED** [review]

File: packages/fiori/src/Bar.js (hunk 1 above)
Comment (Todor-ads) [reply packages/fiori/src/Bar.js]: done

**Review by NHristov-sap — APPROVED** [review]

**Review by fifoosid — APPROVED** [review]

**Comment by vladitasev** [comment]
In general, the commit description should explain what are the reasons for this change with 1-2 sentences. Repeating the PR change has no value.

File: packages/fiori/src/Bar.js [hunk 2]
```diff
@@ -168,6 +170,12 @@ class Bar extends UI5Element {
 		};
 	}
 
+	onBeforeRendering() {
+		// Next two row is specific for IE11. Please remove after stop support and edit css file
+		const content = this.startContent.concat(this.middleContent).concat(this.endContent);
```
[the remark this answers is not in the document]
Comment (vladitasev) [reply packages/fiori/src/Bar.js:175]: Best practice:

[...this.startContent, ...this.middleContent, ...this.endContent].forEach()

**Pull request merged**